sos/report/plugins/lustre.py

```diff
@@ -62,10 +62,11 @@ def setup(self):
         ])
 
         # Server Specific
-        self.get_params(
-            "osd",
-            ["osd-*.*.{mntdev,files*,kbytes*,blocksize,brw_stats}"]
-        )
+        self.get_params("osd", [
+            "osd-*.*.{blocksize,brw_stats,enable_projid_xattr,extent*}",
+            "osd-*.*.{fallocate_*,full_scrub_*,files*,kbytes*,mntdev}",
+            "osd-*.*.{nonrotational,oi_scrub,pdo,stats,read*,write*}"
```

Comment:
minor nit, worth adding a comma at the end, so that if you more, then it's a one line change in the future?